### PR TITLE
New devtools localization and whatsnew videos

### PR DESCRIPTION
**Pull request by captainbrosset**
Two new DevTools videos were published recently: what's new in 102 and a localization demo.
This PR adds both of them to our dev-videos page.

### CODE REVIEW

File: microsoft-edge/dev-videos/index.md [hunk 1]
```diff
@@ -29,6 +29,30 @@ To add a new video:
 -->
 
 
+<!-- ====================================================================== -->
+## DevTools - Use your preferred language in DevTools
```
Comment (vchapel) [inline microsoft-edge/dev-videos/index.md:33]:
```suggestion
## DevTools - Use your preferred coding language in DevTools
```

File: microsoft-edge/dev-videos/index.md [hunk 2]
```diff
@@ -29,6 +29,30 @@ To add a new video:
...
+
+_Published on 9 June 2022._
+
+[![Thumbnail image for the DevTools localization video](./images/devtools-localization.png)](https://www.youtube.com/watch?v=AeF0AvWpUO8)
+
+Microsoft Edge DevTools supports 13 different languages. In this video, the team shows how to choose the language that's most comfortable to you for coding and debugging.
```
Comment (vchapel) [inline microsoft-edge/dev-videos/index.md:39]:
```suggestion
Microsoft Edge DevTools supports 13 different coding languages. In this video, the team shows how to choose the language that's most comfortable to you for coding and debugging.
```

File: microsoft-edge/dev-videos/index.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/dev-videos/index.md:33]: The languages I'm referring to here are not programming languages (which is what the "coding" word you suggest adding makes me think of). These are languages you can translate the user interface to, like French, German, Chinese, etc.

@vchapel any idea on how to make this clearer?

Comment (vchapel) [reply microsoft-edge/dev-videos/index.md:33]: Oh, that is worded correctly then, I misunderstood

**Pull request merged**